test/integration/testdata/gen-flags-then-config.golden

```diff
@@ -1,205 +0,0 @@
----
```

Comment:
Dont need this since it is covered by the TestOutputIncludes test now and doesnt use goldenfiles.